ros/finalarm_control/config/finalarm_gazebo_trajectory_control.yaml

```diff
@@ -1,28 +1,27 @@
-finalarm:
```

Comment:
We remove this namespace so we publish to the main topic, like the physical robot does. Namespacing the topics does make sense though generally.